### PR TITLE
move enableEcmp to subnet

### LINKED ISSUE #2249: Move ECMP option to subnet

### PR DESCRIPTION
**Pull request by hongzhen-ma**
- [ ] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

#### What type of this PR
- Bug fixes
move enableEcmp to subnet

#### Which issue(s) this PR fixes:
Fixes #2249

### CODE REVIEW

File: dist/images/install.sh [hunk 1]
```diff
@@ -1486,6 +1486,8 @@ spec:
                   type: boolean
                 enableLb:
                   type: boolean
+                enableEcmp:
```
Comment (oilbeater) [inline dist/images/install.sh:1489]: crd change should also update helm charts

**Review by oilbeater — CHANGES_REQUESTED** [review]

**Comment by oilbeater** [comment]
Need update the document

**Review by zhangzujian — APPROVED** [review]

File: dist/images/install.sh (hunk 1 above)
Comment (hongzhen-ma) [reply dist/images/install.sh:1489]: The second file is update for helm charts.
The charts in acp can be updated when upgrade kube-ovn image.

**Comment by hongzhen-ma** [comment]
> Need update the document

https://github.com/kubeovn/docs/pull/32

**Review by oilbeater — APPROVED** [review]

**Pull request merged**